stanford-earth/SE3_BLT
# EARTH-118: Adds Research Area content type feature configuration.

#READY FOR REVIEW
#Summary

Added content-type feature for Research Areas
Includes use of viewfield contrib module

#Needed By (Date)

end of current sprint

#Urgency

unknown

#Steps to Test

1. Checkout branch
2. composer update
3. drush cim
4. drush csim local
5. drush features-import-all
6. admin/structure/taxonomy - should now see Research Areas vocabulary
7. node/add - should now see Research Areas content type
